erpnext/stock/doctype/stock_entry/stock_entry.json

```
@@ -386,15 +386,15 @@
   {
    "fieldname": "total_outgoing_value",
    "fieldtype": "Currency",
-   "label": "Total Outgoing Value",
+   "label": "Total Outgoing Value (Consumption)",
    "options": "Company:company:default_currency",
    "print_hide": 1,
    "read_only": 1
   },
   {
    "fieldname": "value_difference",
    "fieldtype": "Currency",
-   "label": "Total Value Difference (Out - In)",
+   "label": "Total Value Difference (In - Out)",
```

```suggestion
   "label": "Total Value Difference (Incoming - Outgoing)",
```
A bit hard to visually cross check right now because instinct was to do LHS - RHS